components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -431,23 +432,53 @@ - (CGFloat)justifiedWidth {
 - (void)scrollUntilSelectedItemIsVisibleWithoutAnimation {
```

Comment:
How is scrolling with animation handled for RTL?

Reply:
It's no different from scrolling in LTR - it's handled above in `setSelected:animated:`.  The fix in this PR is that the position of the `itemView` for RTL was miscalculated.  Here's scroll-to-selected in RTL. The Catalog has a bug with layoutMargins, so I had to hack the top constraint:

![tabbar-rtl-selection-scroll](https://user-images.githubusercontent.com/1753199/60405893-f4ff4500-9b80-11e9-8677-013b69c09651.gif)